### DISCUSSION TITLE
Only one upload request per user per 2 minutes.

### DISCUSSION THREAD

**Discussion by JaLe29** [html2app/discussions#2] [General]
I am trying to test my build, but I am getting this error: `Only one upload request per user per 2 minutes.`,

I waited more than 10 minutes, it looks like the message is wrong or something else is broken.

Any advice?

Or can I pay for some premium?

Thanks

**Comment by yandeu** [comment]
Sorry for that. I have just updated the code:

```ts
// one request per user per 5 minutes
const coeff = 1000 * 60 * 5
const t = new Date(Math.round(new Date().getTime() / coeff) * coeff).toISOString()
const unique = await isUnique('UPLOAD_REQUEST', t + '#' + userHash)
if (!unique)
  return { statusCode: 429, body: JSON.stringify({ err: 'Only one upload request per user per ~5 minutes.' }) }
```

---

No, there is not premium version for now.

**Comment by yandeu** [comment]
There was another bug with the pipeline. (Some breaking changes is missed).
But now it works again :)
